aten/src/ATen/common_with_cwrap.py

```diff
@@ -23,6 +23,10 @@ def parse_arguments(args):
 
 
 def set_declaration_defaults(declaration):
+    if 'raw_string' not in declaration:
```

Comment:
There's also declarations coming from other sources (see run in gen.py). I could edit those too, but this seems easier and more appropriate since it's supposed to set the defaults.